bluespice

```diff
@@ -118,7 +118,7 @@ def main():
         subprocess.call("docker system prune -f", shell=True)
     
     if args.logs:
-        subprocess.call(f'docker exec -ti {dk_container_id} sh -c "tail -f /dev/logs"', shell=True)
+        subprocess.call(f'docker exec -ti {dk_container_id} sh -c "tail -f /data/www/wiki.logs"', shell=True)
```

Comment:
Do we really want to put the log file into `/data/www`? Are we sure this will not be accessible from the web?

Also, why don't we just send all the logs to `stdout`/`stderr`?

Reply:
yea we are NOW sending all the container logs to stdout/stderr, i was also thinking to give the log file access to the user as well

Reply:
Also this file should not be accessed from the web server as the webserver root is /data/www/bluespice, but /data/wiki.logs would also be a better path